pennylane/fourier/coefficients.py

```diff
@@ -256,7 +256,7 @@ def _coefficients_no_filter(f, degree, use_broadcasting):
             nvec = (*nvec, n_ranges[-1])
             sampling_point = [s * n for s, n in zip(spacing, nvec)]
         else:
-            sampling_point = spacing * np.array(nvec)
+            sampling_point = np.squeeze(spacing * np.array(nvec))
```

Comment:
@albi3ro @josh146 this is what was causing the parameter shift demo to fail.

Reply:
nice! I wonder why the demo didn't fail locally for @albi3ro though

Reply:
Maybe she had checked out to a different branch of pennylane which doesn't have the changes to `broadcast_expand`.